config/analytics-v4.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
 return [
-    'property_id' => config('analytics_property_id'),
+    'property_id' => env('ANALYTICS_PROPERTY_ID', 'UA-XXXXXXXXX-X'),
```

Comment:
GA4 properties use 9 numbers only `XXXXXXXXX` and not `UA-XXXXXXXXX ` format. So lets just set a default to `XXXXXXXXX`